combat-trainer.lic

```diff
@@ -1135,8 +1135,11 @@ class SpellProcess
       @equipment_manager.stow_weapon(game_state.weapon_name)
     end
 
-    check_invoke
-    ritual(data)
+    data = update_astral_data(data)
```

Comment:
Are we going to run into the problem where people with multiple characters will download both `combat-trainer` and `common-arcana` at the same time, but not reload `common-arcana`?

Reply:
Yes, that will be an issue. I'd like to avoid that. What is the best way to reload common-arcana? There could be some temporary solution i suppose?

Reply:
My thoughts: I could remove this update, and submit it as a seperate PR

Reply:
It would be nice to get them all as one update.. less work for me :)

Reply:
Is it possible to signal that common-arcana should be reloaded? this might work well with future updates to common scripts

Reply:
One PR is easier, but I'm not sure if it's worth the odd behavior of a wasted cast.

Two PRs is just as easy for me!

Reply:
Lets do two PR's